### ISSUE TITLE
Glob importing not working...

### ISSUE THREAD

**Issue by DrChills** [maoberlehner/node-sass-magic-importer#139]
Not sure what's going wrong here:

this imports fine:
`@import '../../../../global-assets/themes/uk/theme/variables/colors';`

however this:
`@import '../../../../global-assets/themes/uk/theme/variables/*;`
or 
`@import '../../../../global-assets/themes/uk/theme/variables/*.scss';`
Doesnt!

Config:
```
var sass = require('node-sass');
var globImporter = require('node-sass-glob-importer');

...
  var result = sass.renderSync({
      file: options.src,
      importer: globImporter(),
      outputStyle: options.style
  });
```

**Comment by maoberlehner** [comment]
Thanks for your error report. Config looks fine. Is there any specific error message?

Is the full code accessible somewhere?

**Comment by DrChills** [comment]
Thanks for the fast reply!

error: 

```Error: File to import not found or unreadable: E:
                                                 itBucketProject.blogwp-contentglobal-assetsthemesukthemevariables_borders.scss.```
Which seems weird, obviously it's stripped the `/`s 

Environment:
   Windows 10
   node v6.11.3
   npm 5.4.1

Here's my package.json:

```
{
  "scripts": {
    "lint": "npm-run-all lint:*",
    "lint:css": "stylelint wp-content/{themes/plugins}/**/*.scss --cache",
    "lint:js": "eslint wp-content/{themes/plugins}/**/*.js --cache",
    "prepush": "npm-run-all lint:*",
    "commitmsg": "node ./hooks/commit-msg.js",
    "scss:copyassets": "copyfiles -u 2 ../platform.frontend/themes/**/* wp-content/global-assets",
    "scss:compile": "node ./tasks/sass.js",
    "scss:postcss": "postcss ./wp-content/themes/us/compiled/css/sw-main.css -u autoprefixer -u cssnano -r",
    "js:compile": "babel",
    "build": "npm-run-all scss:*",
    "watch": "watch 'npm-run-all scss:*' ./wp-content/themes/**/*.scss"
  },
  "devDependencies": {
    "autoprefixer": "^7.1.4",
    "chokidar-cli": "^1.2.0",
    "copyfiles": "^1.2.0",
    "cssnano": "^3.10.0",
    "eslint": "^4.6.1",
    "eslint-config-airbnb": "12.0.0",
    "eslint-config-airbnb-base": "8.0.0",
    "fs": "^0.0.1-security",
    "husky": "^0.13.3",
    "mkdirp": "^0.5.1",
    "node-fetch": "^1.7.2",
    "node-sass": "4.5.3",
    "node-sass-glob-importer": "^5.0.0-alpha.5",
    "normalize-css": "^2.3.1",
    "npm-run-all": "^4.1.1",
    "path": "^0.12.7",
    "postcss": "^6.0.11",
    "postcss-cli": "^4.1.0",
    "precss": "^2.0.0",
    "stylelint": "^8.1.1",
    "stylelint-declaration-use-variable": "^1.6.0",
    "stylelint-scss": "^2.0.1",
    "watch": "^1.0.2"
  },
  "dependencies": {
    "babel": "^6.23.0",
    "babel-cli": "^6.26.0",
    "babel-preset-es2015": "^6.24.1"
  }
}
```

I'm afraid I can't share the code :-(

**Comment by maoberlehner** [comment]
Ok thanks for the details – I think there might be a problem with Windows directory separators – I'll remove the dust from my Windows PC and check out if I can reproduce the problem ;)

**Comment by maoberlehner** [comment]
I published a new release (`5.0.0-alpha.6`) to npm – this should be fixed. Thank you for reporting this issue and reminding me to consistently test on windows again ;)

**Comment by DrChills** [comment]
Works perfectly, thanks for your hard work! I really appreciate it :-)
